src/tekton_watcher/runs.clj

```diff
@@ -11,8 +11,8 @@
   [taskrun]
   (->> taskrun
        :status
-       :conditions
-       (every? #(not= "Running" (:reason %)))))
+       :completionTime
+       boolean))
```

Comment:
You don't need to convert the return value to boolean since nil is a falsey value.